src/Okta.Idx.Sdk/IIdxClient.cs

```diff
@@ -44,16 +44,18 @@ public interface IIdxClient : IOktaClient
         /// </summary>
         /// <param name="authenticationOptions">The authentication options.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
+        /// <param name="recoveryToken">The recovery token.</param>
         /// <returns>The authentication response.</returns>
-        Task<AuthenticationResponse> AuthenticateAsync(AuthenticationOptions authenticationOptions, CancellationToken cancellationToken = default);
+        Task<AuthenticationResponse> AuthenticateAsync(AuthenticationOptions authenticationOptions, CancellationToken cancellationToken = default, string recoveryToken = null);
```

Comment:
Since this new param is intended to be used during self-service password recovery, I'm not sure if recovery token should be passed to the authenticate method. There was a discussion about this last week, and IIRC Jeff wanted to clarify this with the corresponding team.